terraform/deployments/dev/main.tf

```diff
@@ -6,4 +6,5 @@ module "ggroup-to-splunk-dev" {
   lambda_memory       = 256
   lambda_role_name    = "ggroup_lambda_exec_role"
   lambda_timeout      = 900
+  domain              = "digital.cabinet-office.gov.uk"
```

Comment:
Should we change this dev one to the `cabinet-office.co.uk` domain? Not on this PR but as a tech-debt issue for later.

Reply:
yeah great idea - I'll write a ticket for that now